publish/Folder-parameters.md

```diff
@@ -0,0 +1,38 @@
+
+# Folder-parameters in leaflet
+
+## `geojsonFolder`
+The `geojsonFolder` Tag allows to specify one or more Folders to contain *.geojson or *.json Files to be rendered into the current map.
+
+## `markerFolder` 
+Similarly, the `markerFolder` Tag allows to specify one or more Folders to contain *.md Files with `location` and `mapmarker` Tags in their YAML FrontMatter to be displayed as Markers in the Map. 
+
+Both Folder Paths can start with a 
+- '/' (Slash) based on the Vault Directory or
+- './' relative to the current *.md Document 
+
+By default only Files directly in the Folder are added. 
```

Comment:
I reverted this in https://github.com/javalent/obsidian-leaflet/releases/tag/5.6.0. It defaults to recursion.

Reply:
Ah, ok, so did you implement the behavior proposed by @Sirs0ri? Is it still possible to limit the Recursion Depth using '/'s?

Reply:
That is correct, it just defaults to infinite instead of none

Reply:
updated documentation to document infinite recursion by Default.